.travis.yml

```diff
@@ -0,0 +1,13 @@
+langauge: ruby
+rbenv:
+- 2.5.3
+addons:
+  postgresql: 9.6
+script:
+- bundle exec rails db:{create,migrate} RAILS_ENV=test
+- bundle exec rspec
+deploy:
+  provider: heroku
+  api_key:
```

Comment:
There is a Travis Command Line function that will generate a secure api key for this file. This is the page I referenced previously https://docs.travis-ci.com/user/deployment/heroku/

It also talks about the continuous deployment, but I had trouble getting that to work last time around

Reply:
Yeah, I set continous deployment on Heroku, and making it conditional on passing CI. 

I'm also a little unclear on the need for the api key. We didn't use it for viewing party. I'll take a look at the docs

Reply:
Sounds good! Dione's explanation helped, I did not know about that Heroku-side option. I think that api key may only be needed if Travis is deploying, so it may not be necessary for us at all